dynatrace/environment_v2/problems.py

```diff
@@ -0,0 +1,301 @@
+"""
+Copyright 2021 Dynatrace LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+from enum import Enum
+from requests import Response
+from datetime import datetime
+from typing import Optional, Union, Dict, Any, List
+
+from dynatrace.http_client import HttpClient
+from dynatrace.dynatrace_object import DynatraceObject
+from dynatrace.environment_v2.schemas import ManagementZone
+from dynatrace.environment_v2.metrics import Unit
+from dynatrace.environment_v2.monitored_entities import EntityStub
+from dynatrace.configuration_v1.metag import METag
+from dynatrace.configuration_v1.alerting_profiles import AlertingProfileStub
+from dynatrace.pagination import PaginatedList
+from dynatrace.utils import int64_to_datetime
+
+
+class ProblemService:
+    ENDPOINT = "/api/v2/problems"
```

Comment:
I like this idea, makes the rest very clean and easier to understand what endpoints this implements